### ISSUE TITLE
Too much monitor is being recognized after I set the 'Virtual Machine Platform' enabled

### ISSUE THREAD

**Issue by azul4** [MicrosoftDocs/WSL#1104]
Note that this repo's issue tracker is only for reporting issues with the WSL docs.

Other related issue repo's:

* WSL: https://github.com/Microsoft/BashOnWindows/
* Windows Console: https://github.com/Microsoft/Console/

===========
Hi folks. I have 2 monitors in my desk, one is a notebook and the other one is a monitor itself.

  As all of you might know to use WSL we got to enable 'Virtual Machine Platform' and 'Windows Subsystem for Linux' from 'Windows Facilities On/Off'

  So my problem is that whenever I enable 'Virtual Machine Platform' to use WSL, the monitor wouldn't work appropriately. More precisely, after enabling it, the CPU recognizes a 6 monitors. Obviously I have only 2 monitors so it leads to a situation that mouse cursor would go further to a virtual monitor (which literally doesn't exists.) Plus the 2 monitors that really exists can't append from one to another but only can duplicate, which means it doesn't have any meaning to have a dual monitor for me. Whenever I disable 'Virtual Machine Platform' everything works fine, and don't know where this problem begins.

  Since I am very new to here, writing a issue on github, please let me know what else I have to offer you about my notebook's specifications. Thanks.

**Comment by azul4** [comment]
Well basically I solved myself twice, and here's the solutions:

One was to update my windows version, and I  what was my previous version was, since I didn't expect it to solve my problem. Currently it's version 20H2, OS build 19042.
The other was to install a graphic device driver (Intel Graphics Driver) and the monitor worked properly. 
Unfortunately neither of them do I know what makes the monitor to work appropriately. 

And now I think that it's the problem belongs to Win10's rather WSL's.

**Comment by wideseen** [comment]
In Win10 I had enabled Windows  Subsystem for Linux and installed WSL.
Now extending to my HDMI monitor did not work, it fell back on copy 1->2.
Win10 showed extra virtual monitors in Device Manager and Display Settings
After some panicky deinstalling the WSL, disabling Windows  Subsystem for Linux, removing the virtual monitors, restarts and other non-destructive whatnot, suddenly the monitor extended. OK now.
BUT - no sound on the monitor. No driver reinstall helped.
Then I found this thread. Thankyou!!
The Virtual Machine Platform had apparently been automatically enabled when enabling Windows  Subsystem for Linux; I didn't checkmark it.
And lo-and-behold, disabling this also and the HDMI monitor was back in action.
I go with Cygwin now for Linux in Win.
